bin/enable-executable

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env ruby
+
+require 'fileutils'
+require 'pathname'
+
 puts 'Ensuring test files are executable...'
 
-Dir.glob('**/*test.rb').each do |f|
+test_files = Dir.glob(File.join(__dir__, '../exercises/**/*test.rb'))
+bin_files = Dir.glob(File.join(__dir__, '*'))
+
+(test_files + bin_files).each do |f|
   if File.executable?(f)
     print '.'
   else
-    print "Adding exec bit to #{f}."
-    File.chmod(0744, f)
+    relative_path = Pathname.new(f).relative_path_from(Pathname.pwd)
+    print "\nAdding exec bit to #{relative_path}."
```

Comment:
I feel that this is a matter of semantics. `path` can refer to the path to a file, not just directories. Still, I'd be willing to update it to something like `relative_path_to_f`, but at that point the variable name is semantically equivalent to the right side of its assignment.